packages/components/src/setup.ts

```diff
@@ -9,4 +9,5 @@ export default (): void => {
 
   injectGlobalStyle();
   validatePartialUsage();
+  validateVersions();
```

Comment:
I am wondering this is the best place, perhaps `loadComponentLibrary()` would work better since prefix collisions are checked there.
Also, this will only reliably work when it is executed last, right? So if this version is initialized first, it will miss later ones.
How does this behave in a real application, e.g. porsche.com or even our sample integrations?